src/schema/libraries.js

```diff
@@ -104,7 +104,13 @@ export const resolvers = {
       return status === false;
     },
     async borrowerCheck(parent, args, context, info) {
-      return await resolveBorrowerCheck(parent.agencyId, context);
```

Comment:
er det ikke altid agency der burde tjekkes for borrowerCheck? et enkelt bib kan ikke have borrower check hvis væsenet ikke har det?